sql/lanterndb.sql

```diff
@@ -1,77 +1,114 @@
 -- Definitions concerning our hnsw-based index data strucuture
-
-CREATE FUNCTION hnsw_handler(internal) RETURNS index_am_handler
-	AS 'MODULE_PATHNAME' LANGUAGE C;
+CREATE FUNCTION hnsw_handler (internal)
+    RETURNS index_am_handler
+    AS 'MODULE_PATHNAME'
+    LANGUAGE C;
 
 CREATE ACCESS METHOD hnsw TYPE INDEX HANDLER hnsw_handler;
 
 COMMENT ON ACCESS METHOD hnsw IS 'LanternDB vector index access method. Can be configured to use various strategies such hs hnsw, graph-based, disk-optimized etc.';
 
-
 -- functions
-CREATE FUNCTION ldb_generic_dist(real[], real[]) RETURNS real
-	AS 'MODULE_PATHNAME' LANGUAGE C IMMUTABLE STRICT PARALLEL SAFE;
-	
-CREATE FUNCTION ldb_generic_dist(integer[], integer[]) RETURNS real
-	AS 'MODULE_PATHNAME' LANGUAGE C IMMUTABLE STRICT PARALLEL SAFE;
-	
-CREATE FUNCTION l2sq_dist(real[], real[]) RETURNS real
-	AS 'MODULE_PATHNAME' LANGUAGE C IMMUTABLE STRICT PARALLEL SAFE;
+CREATE FUNCTION ldb_generic_dist (real[], real[])
+    RETURNS real
+    AS 'MODULE_PATHNAME'
+    LANGUAGE C
```

Comment:
Do we have the option of turning off `pgFormatter` formatting for parts of the file (e.g. via a magic-comment.
e.g. in clang-format, any code I write within the magic comments below will not be formatted
```
// clang-format off
[CODE HERE]
// clang-format on
```
We should use a formatter that has such an option and should use such an option for sql statements that are very similar. This allows to more easily tell what is different between similar SQL statements.


For example, in the above, the two `ldb_generic_dist`, and `l2sq_dist` only differ in their argument types and that is easier to quickly verify visually when we have long "unformatted" sql lines.

Reply:
```
 -p | --placeholder RE : set regex to find code that must not be changed.
```
We could use this to achieve that.

Reply:
Ok, great! 
Let's do that for these kind of repeating statements.